Babylon/main.py

```diff
@@ -2,57 +2,70 @@
 # -*- coding:utf-8 -*-
 import importlib.util
 import logging
+import os
+import pathlib
+import sys
 
 import click
 import click_log
-import sys
 
 from .commands import list_commands
 from .groups import list_groups
 from .utils.configuration import Configuration
+from .utils.environment import Environment
 from .utils.logging import MultiLineHandler
-from .utils.solution import Solution
-from .v0 import v0
+from .utils.working_dir import WorkingDir
 
 logger = logging.getLogger("Babylon")
 handler = MultiLineHandler()
 formatter = logging.Formatter('{levelname:>8} - {asctime} | {message}', style='{', datefmt='%Y/%m/%d - %H:%M:%S')
 handler.setFormatter(formatter)
 logger.addHandler(handler)
 
-conf = Configuration(logger)
+config_directory = pathlib.Path(os.environ.get('BABYLON_CONFIG_DIRECTORY', click.get_app_dir("babylon")))
+platform_override = os.environ.get('BABYLON_PLATFORM_OVERRIDE')
```

Comment:
To discuss but overriding platform and deploy files gives to much flexibility and complicate the maintainability (having to prevent action because we are not sure conf files are). Just override the conf_dir should suffice and still allow save.